src/test/java/resources/account.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Account Management test
+
+  Scenario: Account status API Test
+    Given I have tenant as "gorilla"
+    When I call accountStatus endpoint for "835322416" MSISDN, with status of 200
+    Then I should get non empty response
```

Comment:
this is not enough for testing the status. you should check for the expected status.

Reply:
This is must